certora/specs/SocialRecoveryModule.spec

```
@@ -1,3 +1,27 @@
-rule demo() {
-    assert true;
+using SafeHarness as safeContract;
+
+methods {
+    // Safe Functions
+    function safeContract.isModuleEnabled(address) external returns (bool) envfree;
+    function safeContract.disableModule(address, address) external;
+}
+
+// A setup function that requires Safe contract to enabled the Social Recovery
+// Module.
+function setupRequireRecoveryModule() {
```

Nit: not a fan of the name (note it has a different purpose than in the safe token FV spec):

```suggestion
function requireSocialRecoveryModuleEnabled() {
```

---

Updated 👍🏾